docs/configuration/configuration.md

```diff
@@ -247,6 +247,10 @@ kubernetes_sd_configs:
 lightsail_sd_configs:
   [ - <lightsail_sd_config> ... ]
 
+# List of Linode service discovery configurations.
```

Comment:
You need to add this twice in this page: one for the scrapes, one for the alertmanager SD.